setup.py

```diff
@@ -34,8 +34,15 @@ def run(self):
     "datasets>=2.7.0",
 ]
 
+CLEANVISION_REQUIRE = [
+    # Mainly for Datalab's data storage class.
+    # Still some type hints that require datasets
+    "cleanvision==0.3.0",
+]
+
 EXTRAS_REQUIRE = {
     "datalab": DATALAB_REQUIRE,
+    "cleanvision": CLEANVISION_REQUIRE,
```

Comment:
I vote to name this key "image" instead. So for installing all optional dependencies related to image data, future cleanlab users can do: `pip install cleanlab[image]`

I think cleanvision is just one special case of this in the long-term.